frontend/src/components/features/project-menu/ProjectMenuCard.tsx

```diff
@@ -34,26 +30,6 @@ export function ProjectMenuCard({
     setContextMenuIsOpen((prev) => !prev);
   };
 
-  const handlePushToGitHub = () => {
```

Comment:
This is the "Push to Github" button which was not necessary I don't think.

Reply:
I think this would show up _after_ you already pushed and created a PR?

Reply:
Don't believe so but not 100% sure. Originally this only showed up on local setup only...And it seems unnecessary. I'll wait for Stephan to comment.

Reply:
This is found in the bottom right project card, after clicking the three dots. The options are "push to github" or "download zip"

It makes sense to remove it if we already display it in the chat interface